storage_thumbnail/tests/test_thumbnail.py

```diff
@@ -0,0 +1,59 @@
+import base64
+import os
+
+from odoo.addons.component.tests.common import TransactionComponentCase
+from odoo.fields import first
+
+from .models import ModelTest
+
+
+class TestStorageThumbnail(TransactionComponentCase):
+    def setUp(self):
+        super().setUp()
+
+        # Register model inheritance
+        ModelTest._build_model(self.env.registry, self.env.cr)
```

Comment:
@rousseldenis You don't cleanup the registry at the end of your tests..... https://github.com/OCA/search-engine/blob/12.0/connector_search_engine/tests/models_mixin.py#L51

Reply:
Good catch. I forgot teardown

Reply:
@lmignon Done